src/dmd/cparse.d

```diff
@@ -4410,19 +4468,19 @@ final class CParser(AST) : Parser!AST
         /* void __builtin_va_start(__builtin_va_list, ...);
          * The second argument is supposed to be of any type, so fake it with the ...
          */
-        genBuiltinFunc(Id.builtin_va_start, AST.VarArg.variadic);
+        //genBuiltinFunc(Id.builtin_va_start, AST.VarArg.variadic);
 
         /* void __builtin_va_end(__builtin_va_list);
          */
-        genBuiltinFunc(Id.builtin_va_end, AST.VarArg.none);
+        //genBuiltinFunc(Id.builtin_va_end, AST.VarArg.none);
 
         /* struct __va_list_tag
          * {
          *    uint, uint, void*, void*
          * }
          */
-        auto s = new AST.StructDeclaration(Loc.initial, Id.va_list_tag, false);
-        symbols.push(s);
+//        auto s = new AST.StructDeclaration(Loc.initial, Id.va_list_tag, false);
+//        symbols.push(s);
```

Comment:
Since this is generated by importing the `__builtin` module, is this really needed to be commented or should you remove it?

Reply:
I also suggest we remove all this commented code.  At this point, this function contains more commented code than actual code.

Reply:
This PR creates a lot more dead code than just those lines. Rather than confuse this PR, which I wanted to make as simple as I could, my plan is to remove the dead code in a followup PR.